### ISSUE TITLE
Camera API is Deprecated in Android 

### ISSUE THREAD

**Issue by ahmad2smile** [messier16/FullCameraPage#2]
AFAIK, The Camera API is deprecated and you have to use Camera2 in Android and it quite different. You are using it [here](https://github.com/messier16/FullCameraPage/blob/8e5f6c69ea82a7ba6eef97ee792e99b6fca077cc/FullCameraApp.Android/CameraPageRenderer.cs#L36).

Any chance you update it?

**Comment by fferegrino** [comment]
Yep I am aware of that, I stated that in [the README.md](https://github.com/messier16/FullCameraPage#warning), to be honest, I don't see myself updating it right away. But it would be great if you could help me off a bit ;).

**Comment by ahmad2smile** [comment]
I'm actually looking for a implementation of Camera2 when I get it I'll try to send you a PR. Thanks.

**Comment by fferegrino** [comment]
Thank you, it will be great to see this updated to that API.

**Comment by rmawani** [comment]
Hi guys,

In case anybody still needs it, I found a similar solution [here](https://github.com/UNIT-23/Xam-Android-Camera2-Sample) using Camera2 instead of Camera. It's not ContentPage based but does the trick nicely. 

Works well for me so hope it helps others as well!
